public/app/api/clients/folder/v1beta1/index.ts

```diff
@@ -5,4 +5,4 @@ export const folderAPIv1beta1 = generatedAPI.enhanceEndpoints({});
 export const { useGetFolderQuery } = folderAPIv1beta1;
 
 // eslint-disable-next-line no-barrel-files/no-barrel-files
-export { type Folder } from './endpoints.gen';
+export { type Folder, type GetFolderChildrenApiResponse } from './endpoints.gen';
```

Comment:
```suggestion
export { type Folder, type FolderList } from './endpoints.gen';
```
This type seems to be an alias for `FolderList`, so better export that one as it's more clear what it is.